lib/Mojo/Util.pm

```diff
@@ -183,6 +184,30 @@ sub monkey_patch {
   *{"${class}::$_"} = set_subname("${class}::$_", $patch{$_}) for keys %patch;
 }
 
+sub network_contains {
```

Comment:
I'm not sure if I understand what this function does by looking at the name. Could it be cidr_contains($cidr, $addr), ip_within($addr, $cidr) or something else that implies that it's working with IP addresses?

Reply:
I find the name descriptive enough.

Reply:
A colleague suggested `cidr_contains` as well and I could be convinced, but I personally think that's too "wonky" of a name. You have to know what it is to know what it is in a way. `network_contains` is similarly accurate and more descriptive to the outside reader imo.

Reply:
I suggested ip_within(), since NetAddr::IP has a “within” method. Not a deal breaker for me, but I think it would be nice if we could steal a name from somewhere else.

Reply:
There is no consistency on CPAN. Net::IP calls it "overlaps" with some exported constants to check for different types of overlap, Net::IP::Lite(::Net) calls it "contains" which is where I got my name.